Makefile

```diff
@@ -1,14 +1,19 @@
+SHELL := /bin/bash
+
 default: build
 
 .PHONY: init-examples
 init-examples:
```

Comment:
I'm using this makefile target to test my local changes. Can you introduce a new one for GH action instead of modifying ?

Reply:
But you can test your changes now the same as before. Running `make init-examples` and then `terraform init` from within one of the examples will work.